wenet/transformer/asr_model.py

```diff
@@ -89,6 +94,9 @@ def forward(
             text: (Batch, Length)
             text_lengths: (Batch,)
         """
+        if self.lfmmi_dir != '':
```

Comment:
Why load it in forward? I think we should load it in construction.

Reply:
Then there is no need to use hasattr when loading resource

Reply:
> Why load it in forward? I think we should load it in construction.

because i need to decorate it with torch.jit.ignore, i dont want to decorate whole forward function with torch.jit.ignore

Reply:
now we move load_mmi_resource into __init__ and remove hasattr judgement